app/services/twilio_service.rb

```diff
@@ -26,8 +30,9 @@ def send_sms(params)
         body: body,
         to: to
       )
-    rescue => e
-      Rails.logger.error("send SMS failed: #{e}")
+    rescue => error
+      Rails.logger.error("send SMS failed: #{error}")
```

Comment:
I would rather see a bugsnag about it but I don't know that we're set up for that...